content/doc/pipeline/declarative/Reporting-test-results-and-storing-artifacts.adoc

```diff
@@ -0,0 +1,68 @@
+If you have a test runner (such as maven surefire, or many of the
+javascript runners) chances are it can output the junit/xml style of
+test result reports. If so, recording them is easy:
+
+[source,groovy]
```

Comment:
`[source,groovy]` is okay here because this is a code snippet, but it might actually be interesting to modify this